src/cc_catalog_airflow/dags/provider_api_scripts/jamendo.py

```diff
@@ -0,0 +1,322 @@
+"""
+Content Provider:       Jamendo
+
+ETL Process:            Use the API to identify all CC-licensed images.
+
+Output:                 TSV file containing the image, the respective
+                        meta-data.
+
+Notes:                  https://api.jamendo.com/v3.0/tracks/
+                        No rate limit specified.
```

Comment:
The issue says:
> Rate Limit: 35,000 requests per month for non-commercial apps

Reply:
Thank you! Adding the rate limit and some more notes on Jamendo Music.